google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -124,6 +129,10 @@ public class Publisher implements PublisherInterface {
   private final GrpcCallContext publishContext;
   private final GrpcCallContext publishContextWithCompression;
 
+  private final boolean enableOpenTelemetryTracing;
+  private final OpenTelemetry openTelemetry;
+  private PubsubTracer tracer = new BasePubsubTracer();
```

Comment:
The `BaseApiTracer` in Gax was created before Java 8 introduced `default` methods in interfaces. We actually plan to remove it since we made all methods in `ApiTracer` as `default` now. I think this can be an anonymous inner class so we don't have to introduce a new public class `BasePubsubTracer`.

Reply:
As noted below, I've removed the `PubsubTracer` interface.